package.js

```diff
@@ -16,10 +16,5 @@ Package.onUse(function(api) {
     'peerlibrary:data-lookup@0.1.0'
   ]);
 
-  api.use([
-    'akryum:vue-router@0.2.2',
-    'akryum:vue-router2@0.1.0'
-  ], {weak: true});
```

Comment:
Why removing this? Leave it as weak dependency.

Reply:
I just thought of removing something that will not really be a dependency on this package, since it works fine with just the Vue Router directly from npm.

The problem in reality is just the [condition](https://github.com/meteor-vue/blaze-integration/pull/10/files#diff-7a9076d6d94e62c13d641aa71f19ae8eL7) because when I removed the package dependency in the [Guide](https://github.com/meteor-vue/guide/pull/4) to use only the Vue Router a problem in RouterLink.

Reply:
So the issue is that NPM package will not populate `Package['akryum:vue-router']`. So if you are using that as a check, then you have to have a Meteor dependency, no?

Reply:
In this case I agree, instead of removing these conditions maybe we should keep them and add another to check if the NPM package is installed.

##### I was just analyzing the context of the Vue Router and in the case as you said this package can also be used with Blaze Router

Reply:
https://github.com/meteor-vue/blaze-integration/pull/10/commits/00f18661ec1d9c99586f8510051b0c46fa35395f I added the dependency but I kept the removal of the link to import the [router-link.js](https://github.com/meteor-vue/blaze-integration/blob/master/router-link.js), avoid breakage If the user is not installed the package